src/control/events/ras.go

```diff
@@ -222,6 +238,57 @@ func (evt *RASEvent) FromProto(pbEvt *sharedpb.RASEvent) (err error) {
 	return
 }
 
+// PrintRAS generates a string representation of the event consistent with
+// what is logged in the data plane.
+func (evt *RASEvent) PrintRAS() string {
```

Comment:
I wonder if it makes sense to put this helper in the `dmg/pretty` package. I don't feel too strongly either way, though.

Reply:
I'd agree with putting it in dmg/pretty as long as it's only intended to be used in dmg.

Reply:
it's used to log the messages and will eventually provide a structured output to syslog so I would prefer to leave it here for the moment (it's not used in dmg)